src/ImageSupport/Helpers/ImageResizeHelper.cs

```diff
@@ -0,0 +1,103 @@
+﻿using LT.DigitalOffice.ImageSupport.Constants;
+using LT.DigitalOffice.ImageSupport.Helpers.Interfaces;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Processing;
+using Svg;
+using System;
+using Drawing = System.Drawing;
+using System.IO;
+using System.Threading.Tasks;
+using LT.DigitalOffice.Kernel.Constants;
+
+namespace LT.DigitalOffice.ImageSupport.Helpers
+{
+  public class ImageResizeHelper : IImageResizeHelper
+  {
+    #region private methods
+
+    private Task<(bool isSuccess, string resizedContent, string extension)> SvgResize(string inputBase64, string extension, int resizeMinValue)
+    {
+      return Task.Run(() =>
+      {
+        try
+        {
+          byte[] byteString = Convert.FromBase64String(inputBase64);
+
+          using (MemoryStream ms = new MemoryStream(byteString))
+          {
+            SvgDocument svgDocument = SvgDocument.Open<SvgDocument>(ms);
+            Drawing.Bitmap image = svgDocument.Draw();
+
+            if (image.Width > resizeMinValue || image.Height > resizeMinValue)
+            {
+              double maxSize = Math.Max(image.Width, image.Height);
+
+              int ratio = Convert.ToInt32(Math.Ceiling(maxSize / resizeMinValue));
+
+              Drawing.Bitmap newImage = new Drawing.Bitmap(image.Width / ratio, image.Height / ratio);
+              Drawing.Graphics.FromImage(newImage).DrawImage(image, 0, 0, image.Width / ratio, image.Height / ratio);
+
+              Drawing.ImageConverter converter = new Drawing.ImageConverter();
+
+              byteString = (byte[])converter.ConvertTo(newImage, typeof(byte[]));
+              extension = ImageFormats.png;
+
+              return (isSuccess: true,
+                resizedContent: Convert.ToBase64String(byteString),
+                extension: extension);
+            }
+            else
+            {
+              return (isSuccess: true, resizedContent: null, extension: extension);
+            }
+          }
+        }
+        catch
+        {
+          return (isSuccess: false, resizedContent: null, extension: extension);
+        }
+      });
+    }
+
+    private Task<(bool isSuccess, string resizedContent, string extension)>  OtherFormatsResize(string inputBase64, string extension, int resizeMinValue)
```

Comment:
maybe better BaseResize or smth like it, i just don't know which are other

Reply:
Yes, it will be better